sdk/python/feast/repo_config.py

```diff
@@ -54,7 +54,7 @@ class RepoConfig(FeastBaseModel):
     """ Repo config. Typically loaded from `feature_store.yaml` """
 
     registry: Union[StrictStr, RegistryConfig] = "data/registry.db"
-    """ str: Path to metadata store. Can be a local path, or remote object storage path, e.g. gcs://foo/bar """
+    """ str: Path to metadata store. Can be a local path, or remote object storage path, e.g. a GCS URI """
```

Comment:
Is there a reason we can't have an example like `gs://my_bucket/my_path`?

Reply:
I assume there is a way but I spent way too long trying things and googling things to get the literal colon to show up instead of it getting interpreted as a special character like the colon after str. So eventually I gave up and just wrote GCS URI :)

Reply:
Would you mind creating a card to track this then? Perhaps a card to ensure that configurations come with a real example?